nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -58,6 +58,13 @@ let
   # latter case it makes one last attempt at importing, allowing the system to
   # (eventually) boot even with a degraded pool.
   importLib = {zpoolCmd, awkCmd, cfgZfs}: ''
+    for o in $(cat /proc/cmdline); do
+      case $o in
+        zfs_force|zfs_force=1)
```

Comment:
Nit: If you're going to support more than one option, then might as well allow `zfs_force=y` as well.

Reply:
Fair enough